lib/miq_automation_engine/engine/miq_ae_method_service/miq_ae_service_model_base.rb

```diff
@@ -159,12 +159,13 @@ def self.expose(*args)
       options = args.last.kind_of?(Hash) ? args.pop : {}
       raise ArgumentError, "cannot have :method option if there is more than one method name specified" if options.key?(:method) && args.length != 1
 
+      user = User.current_user
```

Comment:
@lfu We need to raise an error here if the user is not set.

Reply:
It should not happen but in case someone stumbles upon it from a simulation or other code paths we should be able to catch it.